Change CD Visibility documentation based on service-centric view (#24157)
DataDog/documentation: content/en/continuous_delivery/search/_index.md

CD Visibility is in private beta. To request access, complete the form.
{{< /callout >}}

## Deployments

To see an overview of your deployments, navigate to [**Software Delivery** > **CD Visibility** > **Deployments**][1].

The [**Deployments** page][1] shows stats aggregated by services and environments over the selected time frame, as well as the status of the latest deployment execution. Use this page to see all your service deployments and get a view of their health.
The metrics shown include the number of executions and failures, the failure rate, the median duration, and the 95th percentile duration. This information reveals which deployments have a higher probability of failure and which deployments are taking the most time to be executed. The effect of the latest changes can be seen by checking the status, revision and time of the last deployment result.

> <div class="alert alert-info">Deployments without services configured and partial deployment executions are excluded from the statistics aggregation of the Deployments page. You can search for these deployments in the Deployment Executions page: <code>@deployment.partial_deployment:* OR -@deployment.service:*</code>.</div>

Reviewer: In the first sentence, is it deployments with partial deployment executions that are excluded or deployments without partial deployment executions that are excluded?

Author: There are two types of deployment excluded from the page:
1. Partial deployments (i.e. only a subset of steps were executed for a deployment)
2. Deployments without service configured (they might or might not have a service, and in the latter case they are excluded from the aggregated stats)

Reviewer: @andrea-mosk Thanks for the explanation! Does this work?

> Deployments with no services configured and those with partial deployment executions are excluded from the statistics aggregation of the Deployments page.

Author: Yeah looks good, I would just remove "those that":

> Deployments with no services configured and partial deployment executions are excluded from the statistics aggregation of the Deployments page.

Author: Changed 😃

{{< img src="/continuous_delivery/search/deployments_2.png" text="The Deployments page in Datadog" style="width:100%" >}}

If you have different ways of deploying a service to an environment, you can expand the deployment rows to see stats further filtered by deployment name.

The **Deployment** page provides you high-level information, including:

- An overview of the health of the different services and environments, with aggregated stats.
- A window for spotting and fixing immediate, urgent issues like broken deployments in production.

> - How each service deployment was executed, over time, and with what results and trends.

Reviewer: ```suggestion
- How each service deployment was executed, over time, and the results and trends.
```
